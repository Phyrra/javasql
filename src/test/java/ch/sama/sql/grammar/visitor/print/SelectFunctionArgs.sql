select coalesce(field1, field2)
from table