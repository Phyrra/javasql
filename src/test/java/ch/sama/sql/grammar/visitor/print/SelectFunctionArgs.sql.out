sql
	statement
		select
			value
				primary
					function
						primary
							field
								field1
						primary
							field
								field2
		from
			source
				table
					table
